<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StandardItemWindow</class>
 <widget class="QMainWindow" name="StandardItemWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableView{

	
	selection-color: rgb(170, 170, 255);	
	selection-background-color: qlineargradient(x1: 0, y1: 0, x2: 0.5, y2: 0.5,
                                stop: 0 #FF92BB, stop: 1 white);

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>TableView</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>PlainTextEdit</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1082</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpen"/>
   <addaction name="actSave"/>
   <addaction name="actModelData"/>
   <addaction name="separator"/>
   <addaction name="actAppend"/>
   <addaction name="actInsert"/>
   <addaction name="actDelete"/>
   <addaction name="separator"/>
   <addaction name="actAlignLeft"/>
   <addaction name="actAlignCenter"/>
   <addaction name="actAlignRight"/>
   <addaction name="actFontBold"/>
   <addaction name="separator"/>
   <addaction name="actExit"/>
  </widget>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/open.png</normaloff>:/icon/icon/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actSave">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/save.png</normaloff>:/icon/icon/save.png</iconset>
   </property>
   <property name="text">
    <string>保存文件</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
  </action>
  <action name="actAppend">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/add2.png</normaloff>:/icon/icon/add2.png</iconset>
   </property>
   <property name="text">
    <string>添加行</string>
   </property>
   <property name="toolTip">
    <string>添加一行</string>
   </property>
  </action>
  <action name="actInsert">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/insert.png</normaloff>:/icon/icon/insert.png</iconset>
   </property>
   <property name="text">
    <string>插入行</string>
   </property>
   <property name="toolTip">
    <string>插入一行</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/jian2.png</normaloff>:/icon/icon/jian2.png</iconset>
   </property>
   <property name="text">
    <string>删除行</string>
   </property>
   <property name="toolTip">
    <string>删除一行</string>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/exit5.png</normaloff>:/icon/icon/exit5.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actModelData">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/prelook.png</normaloff>:/icon/icon/prelook.png</iconset>
   </property>
   <property name="text">
    <string>数据模型预览</string>
   </property>
   <property name="toolTip">
    <string>数据模型预览</string>
   </property>
  </action>
  <action name="actAlignLeft">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/T-left3.png</normaloff>:/icon/icon/T-left3.png</iconset>
   </property>
   <property name="text">
    <string>居左</string>
   </property>
   <property name="toolTip">
    <string>文字左对齐</string>
   </property>
  </action>
  <action name="actAlignCenter">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/T-zhon3.png</normaloff>:/icon/icon/T-zhon3.png</iconset>
   </property>
   <property name="text">
    <string>居中</string>
   </property>
   <property name="toolTip">
    <string>文字居中</string>
   </property>
  </action>
  <action name="actAlignRight">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/T-right3.png</normaloff>:/icon/icon/T-right3.png</iconset>
   </property>
   <property name="text">
    <string>居右</string>
   </property>
   <property name="toolTip">
    <string>文字右对齐</string>
   </property>
  </action>
  <action name="actFontBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/icon/icon/fontAdd2.png</normaloff>:/icon/icon/fontAdd2.png</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>字体加粗</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
